tests/Feature/Booking/BookingTest.php

```diff
@@ -1,17 +1,27 @@
 <?php
 
-namespace Tests\Feature\Position;
+namespace Tests\Feature\Booking;
 
-use App\Booking;
+use App\Modules\Bookings\Booking;
 use App\Position;
 use Carbon\Carbon;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
 use Tests\TestCase;
+use Tests\Unit\Booking\BookingsTestHelper;
 
 class BookingTest extends TestCase
 {
-    use RefreshDatabase, MakesGraphQLRequests;
+    use RefreshDatabase, MakesGraphQLRequests, BookingsTestHelper;
```

Comment:
`RefreshDatabase` is fairly heavy on the overhead, would using `use DatabaseTransactions` be better?

Reply:
The thing I like is that RefreshDatabase gives a clean state each time, especially as some tests are against the DB. All of the SQL stuff is done in memory, which might help with performance.

Reply:
DatabaseTrascations does revert everything after each test though? Unless we have good reason to switch between the two, I think we should just stick with one

Reply:
I think we do RefreshDatabase in every test ?

Reply:
We do use it in every test, yes - just wondering if it's better to convert to DatabaseTransactions to lower the overhead once the test suite gets big?

Reply:
Agreed to keep consistency with RefereshDatabase at the moment